tests/e2e/tests/02-enable-serve-via-traffic/02-send-traffic-large.yaml

```
@@ -0,0 +1,5 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - command: ./send_traffic.sh 'http://target-deployment.$NAMESPACE.svc.cluster.local/bytes/1048576'
+    namespaced: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)